recipe/meta.yaml

```diff
@@ -10,26 +10,28 @@ source:
   sha256: 54905a297a40f6d8b23b94b6652d622ad4756a2b14fb83c0aa76473254d4dd7b
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
+  skip: True  # [py<37]
```

Comment:
what if someone has python 3.7.1 ?

Reply:
Eh, yeah. I guess we need a `run_constrained` with `python >=3.7.2`

Reply:
Done